packages/database/migration/20260706000001_add_feedback_source_directory_workspace_fk/migration.sql

```
@@ -0,0 +1,45 @@
+    orphan_count := orphan_count + 1;
+    RAISE WARNING 'Deleted orphaned FeedbackSource % (name: %, workspaceId: %, feedbackDirectoryId: %) without directory-workspace assignment',
+      orphan."id", orphan."name", orphan."workspaceId", orphan."feedbackDirectoryId";
+  END LOOP;
+
+  IF orphan_count > 0 THEN
+    RAISE WARNING 'Deleted % orphaned FeedbackSource rows before adding the composite FK', orphan_count;
+  END IF;
+END $$;
+
+-- DropIndex
+DROP INDEX "FeedbackSource_feedbackDirectoryId_idx";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)